### PR TITLE
Refactor CC files into CC package

### PR DESCRIPTION
**Pull request by kyguy**
### Type of change

- Refactoring

### Description

Moves the remaining Cruise Control classes of the model package into the `cruisecontrol` subpackage.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/CruiseControl.java
Comment (scholzj) [inline cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/CruiseControl.java:1]: We keep the models in the `model` package. We move there mainly the secondary claases related to a single purpose. So I would keep this class as it was and move only the CruiseControlConfiguration.

Comment (kyguy) [reply cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/CruiseControl.java:1]: A lot more underwhelming than I originally thought :D

Comment (scholzj) [reply cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/CruiseControl.java:1]: Small PRs are good PRs 😉

**Review by fvaleri — APPROVED** [review]

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Pull request merged**